test/utils/Catalyst.spec.ts

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import { HTTPProvider } from 'eth-connect'
+import { getCatalystFromProvider } from '../../src/utils'
+import 'isomorphic-fetch'
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+describe('Catalyst server list', () => {
+  it('loads the catalysts from mainnet', async () => {
+    const provider = new HTTPProvider('https://rpc.decentraland.org/mainnet')
```

Comment:
Same comment as in previous PR, I think we should be using mocks rather than hitting actual endpoints in our tests.

Reply:
Code deleted